src/Sentry.Unity/Integrations/SessionIntegration.cs

```diff
@@ -19,8 +19,6 @@ public void Register(IHub hub, SentryOptions options)
             return;
         }
 
-        options.DiagnosticLogger?.LogDebug("Registering Session integration.");
```

Comment:
The [hub already logs](https://github.com/getsentry/sentry-dotnet/blob/e72ca08d3b0d25cf09690090cd09187c235922b1/src/Sentry/Internal/Hub.cs#L83) which integration get registered. This is redundant.